README.md

```diff
@@ -777,13 +777,13 @@ mod tests {
 
```

Comment:
Comment to other part of README:

Update the example project's deps (for e.g. `cosmwasm-std = "1.2.5"`) to the proper version. Also let's make sure that this corresponds with the `sylvia-template`